src/web/routes.rs

```diff
@@ -142,6 +142,11 @@ pub(super) fn build_routes() -> Routes {
             super::rustdoc::RustLangRedirector::new(redirect),
         );
     }
+    // redirect proc-macro to proc_macro
+    routes.internal_page(
```

Comment:
Couldn't this instead be put with the [rest of the rust-lang redirects](https://github.com/rust-lang/docs.rs/blob/master/src/web/routes.rs#L6-7)?

Reply:
No, because those hard-code that the source and destination are the same. This redirects `/proc-macro` to /proc_macro`.